package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "material-ui-json-schema-viewer",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/taskcluster/material-ui-json-schema-viewer",
+  "license": "MIT",
+  "scripts": {
+    "build": "neutrino build",
+    "start": "neutrino start",
+    "lint": "neutrino lint",
+    "test": "neutrino test",
+    "styleguide": "styleguidist server",
+    "styleguide:build": "styleguidist build"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.7.4",
+    "@babel/plugin-proposal-class-properties": "^7.7.4",
+    "@babel/preset-env": "^7.7.4",
+    "@babel/preset-react": "^7.7.4",
+    "@neutrinojs/jest": "^8.3.0",
+    "@neutrinojs/react-components": "^8.3.0",
+    "@neutrinojs/standardjs": "^8.3.0",
+    "babel-loader": "^8.0.6",
+    "neutrino": "^8.3.0",
+    "prop-types": "^15.7.2",
```

Comment:
`prop-types` is used in the source code but is not guaranteed to be in the application that is importing this library. We should probably have this in the list of `dependencies` to make sure it is installed.

Reply:
Moved `prop-types` to `dependencies` :)